wikimedia/wikipedia-ios
# Show article description box only where appropriate

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T279096

### Notes
* I believe this snuck in in [this PR](https://github.com/wikimedia/wikipedia-ios/pull/3880/). I tried to add a note to help future developers understand what this function does, to future proof this common problem.

### Test Steps
In English:
1. Tap the edit button of a first section of an article w/ an existing description. Should see the article description dialog box.
2. Tap the edit button of the second section of an article w/ an existing description. Should NOT see the box.
3. Tap the edit button of a first section of an article without an existing description Should NOT see the box.
4. Tap the edit button of the second section of an article without an existing description. Should see the box.

Repeat steps 1-4 in a non-English language.
